File: Python/Logic-1/sorta_sum.py
"""

Given 2 ints, a and b, return their sum. However, sums in the range 10..19 inclusive, are forbidden, so in that case just return 20.


sorta_sum(3, 4) → 7
sorta_sum(9, 4) → 20
sorta_sum(10, 11) → 21

@author unobatbayar
"""

def sorta_sum(a, b):
  sum = a + b
  if (sum >= 10) and (sum <= 19):
    return 20
  return sum

File: CodingBat/Java/Logic-1/sortaSum.java
/*
Given 2 ints, a and b, return their sum. However, sums in the range 10..19 inclusive, are forbidden, so in that case just return 20.


sortaSum(3, 4) → 7
sortaSum(9, 4) → 20
sortaSum(10, 11) → 21
*/

class sortaSum{
    public int sortaSum(int a, int b) {
        int sum = a + b;
        if (sum >= 10 && sum <= 19) return 20;
        return sum;
      }      
}